CodeReview/src/CodeReviewManager.h

```diff
@@ -28,19 +28,41 @@
 
 #include "codereview_api.h"
 
+#include "VersionControlUI/src/nodes/DiffFrame.h"
+
 #include "nodes/CommentedNode.h"
 
 namespace CodeReview {
 
+using DiffFrames = QList<VersionControlUI::DiffFrame*>;
+
+using OrderedDiffFrames = QList<VersionControlUI::DiffFrame*>;
```

Comment:
During the meeting we agreed to introduce some special names to avoid getting the method signatures too confusing.